app/views/travels/_form.html.erb

```diff
@@ -1,57 +1,187 @@
-<%= form_with(model: travel, local: true) do |form| %>
-  <% if travel.errors.any? %>
+<div>
```

Comment:
It seems that we can make the whole form easier to work with by checking if the traveller is using car or other means of transportation and showing fields that based on the selection

Reply:
added functionality to hide and show transportation details depending on transportation type. Details are displayed for airplanes, trains, and buses.